docs/src/layouts/MainLayout.vue

```diff
@@ -243,27 +108,25 @@ $hide-social-links-viewport: 862px;
   }
 }
 
-// remove some children just before xs
-.toolbar-menu-items {
-  .q-btn:nth-last-child(-n+3) {
-    @media screen and (max-width: $adjust-header-viewport) {
-      display: none;
-    }
-  }
+.doc-left-drawer {
+  overflow: inherit !important;
 }
 
-.social-links {
-  @media screen and (max-width: $hide-social-links-viewport) {
-    display: none;
-  }
+.mainlayout {
+  font-family: $lp-font-family;
 }
 
-body {
-  font-family: $lp-font-family;
+.drawer-scroll-area {
+  height: 100%;
+}
+
+.layout-countdown {
+  background: linear-gradient(45deg, #e6f1fc 25%, #c3e0ff 25%, #c3e0ff 50%, #e6f1fc 50%, #e6f1fc 75%, #c3e0ff 75%, #c3e0ff);
+  background-size: 40px 40px;
 }
 
-.menu-item > *:hover {
-  background-color: rgba($lp-primary, 0.08);
+.app-menu .q-item {
+  font-size: 14px;
```

Comment:
Didn't we say to add a `font-size-14` and add it as `headerClass` when creating the `QExpansionItem`?

Reply:
For some reason, just adding text-size-14 does not affect the font-size (I tried contentClass too). Also, even if it worked then I'll have to add it to the children also. But if I add as I did then the children inherit it.

Reply:
What's the reason? CSS is deterministic, even if confusing: if there's "some reason", you should take the time to find it out
That's how you grow in knowledge

Reply:
My bad there, there was some other css interferring, I just didn't investigate further because I preferred the solution of styling .app-menu, so that children would inherit too.